ADOPTERS.md

```diff
@@ -5,7 +5,7 @@
 | [Roadie](https://roadie.io)                                              | [@dtuite](https://github.com/dtuite)                                                                                                                                                                         | Hosted, managed Backstage with easy set-up                                                                                                                                                                                                                                                                                                           |
 | [Roku](https://www.roku.com)                                             | [@timurista](https://github.com/timurista)                                                                                                                                                                   | Initial work on Cloud engineering service platform.                                                                                                                                                                                                                                                                                                  |
-| [SDA SE](https://sda.se)                                                 | [@Fox32](https://github.com/Fox32)                                                                                                                                                                           | Central place for developing and sharing services in our insurance ecosystem.                                                                                                                                                                                                                                                                        |
+| [SDA SE](https://sda.se)                                                 | [@backstage/sda-se-reviewers](https://github.com/orgs/backstage/teams/sda-se-reviewers)                                                                                                                      | Central place for developing and sharing services in our insurance ecosystem.                                                                                                                                                                                                                                                                        |
```

Comment:
This group is not browsable for non members so it has to be one/more users :)

Reply:
@jhaals ah damm - will update it